README .es.md

```diff
@@ -0,0 +1,48 @@
+# Zoom Recordings Downloader
+
+> Este [README en inglés](README.md)
+
+Una librería Python para descargar las grabaciones de cuentas Zoom.us
+
+## Arquitectura y requisitos
+
+Para acceder automáticamente a las grabaciones para descarga, se debe disponer de una cuenta Zoom con privilegios developer en la nube Zoom. Se podrá entonces descargar las grabaciones de todas las salas Zoom a las que la cuenta utilizada tenga acceso. 
```

Comment:
Cambiaría por "Para acceder automáticamente a la descarga de grabaciones, ..." o podría también "Para poder descargar las grabaciones utilizando este utilitaro, ..."

Reply:
Ok, that's better. And let's also change the wording in English.